worldengine/plates.py

```diff
@@ -36,25 +36,30 @@ def generate_plates_simulation(seed, width, height, sea_level=0.65,
     return hm, pm
 
 
-def _plates_simulation(name, width, height, seed, temps, humids, num_plates=10,
-                       ocean_level=1.0, step=Step.full(),
-                       verbose=get_verbose()):
+def _plates_simulation(name, width, height, seed, temps=
+                       [.874, .765, .594, .439, .366, .124], humids=
+                       [.941, .778, .507, .236, 0.073, .014, .002], gamma_curve=1.25,
+                       curve_offset=.2, num_plates=10, ocean_level=1.0,
+                       step=Step.full(), verbose=get_verbose()):
     e_as_array, p_as_array = generate_plates_simulation(seed, width, height,
                                                         num_plates=num_plates,
                                                         verbose=verbose)
 
-    world = World(name, width, height, seed, num_plates, ocean_level, step, temps, humids)
+    world = World(name, width, height, seed, num_plates, ocean_level, step, temps,
+                  humids, gamma_curve, curve_offset)
     world.set_elevation(numpy.array(e_as_array).reshape(height, width), None)
     world.set_plates(array_to_matrix(p_as_array, width, height))
     return world
 
 
-def world_gen(name, width, height, seed, temps, humids, num_plates=10, ocean_level=1.0,
-              step=Step.full(), fade_borders=True, verbose=get_verbose()):
+def world_gen(name, width, height, seed, temps=[.874, .765, .594, .439, .366, .124],
+              humids=[.941, .778, .507, .236, 0.073, .014, .002], num_plates=10,
```

Comment:
could we use a constant to avoid repeating the default values for humids and temps?